ci/jira_query.py

```diff
@@ -24,13 +24,16 @@
 # past.
 VALID_COMPONENTS = ('build', 'ci', 'doc', 'gha', 'test')
 
+# Expected ticket prefix.
+VALID_TICKET_PREFIX = ('DAOS', 'CORCI', 'SRE')
+
 # 10044 is "Approved to Merge"
 # 10045 is "Required for Version"
 FIELDS = 'summary,status,labels,customfield_10044,customfield_10045'
 
-# Expected values for Status.  Tickets which are closed should not be being worked on, and tickets
+# Excluded values for Status.  Tickets which are closed should not be being worked on, and tickets
 # which are Open or Reopened should be set to In Progress when being worked on.
-STATUS_VALUES_ALLOWED = ('In Review', 'In Progress')
+STATUS_VALUES_NOT_ALLOWED = ('Open', 'Reopened', 'To Do', 'Resolved')
```

Comment:
We discussed it in the gatekeeping channel and Jeff and Kelsey agreed to this blacklist.  Feel free to merge in these changes into your ticket.  That probably makes more sense rather than having two PRs.

Reply:
I saw.  I just hadn't personally gone through the process of looking at all the possible values here.